man/udev.xml

```diff
@@ -685,6 +685,62 @@ SUBSYSTEM=="net", OPTIONS="log_level=debug"</programlisting></para>
           </varlistentry>
         </variablelist>
 
+        <para>The order in which expressions in each rule are processed is
```

Comment:
Isn't the order an impl. detail that doesn't need to be exposed? If it's documented, people may start relying on it, hindering possible future changes. Maybe a note like "The order in which expressions in each rule are processed does not have to follow the order in which they are written." would be enough?